src/index.js

```diff
@@ -12,11 +12,14 @@ export const renderReact = (name, component) => hypernova({
   },
 
   client() {
-    const { node, data } = load(name);
+    const payloads = load(name);
 
-    if (node) {
-      const element = React.createElement(component, data);
-      ReactDOM.render(element, node);
+    if (payloads) {
+      Array.prototype.map.call(payloads, (payload) => {
```

Comment:
Whoops, you're totally right - i misread :-) in this case I think we can just do `payloads.map` then